### PR TITLE
Minor bugfixes

### PR DESCRIPTION
**Pull request by ralf-meyer**
A few bugs that I encountered during my geometry optimization investigations. The only important change is in `GetCMDists()` which now includes a `try except` block to cover the case of distance matrices that do not satisfy the triangle inequality. This workaround would not be needed if we fixed the `Metrize()` function, which I am not comfortable doing alone since it impacts core functionality for conformer construction.

### CODE REVIEW

File: molSimplify/Scripts/inparse.py [hunk 1]
```diff
@@ -483,7 +483,7 @@ def cleaninput(args):
                 args.keepHs[i] = checkTrue(s)
     # parse FF settings:
     # if no FF opt is requested, turn off
-    if args.ffoption[0].lower() == ('n' or 'no'):
```
Comment (naveenarun) [inline molSimplify/Scripts/inparse.py:486]: good catch 😭😂

**Comment by naveenarun** [comment]
Everything looks good!

File: molSimplify/Scripts/inparse.py (hunk 1 above)
Comment (naveenarun) [reply molSimplify/Scripts/inparse.py:486]: interestingly `('n' or 'no')` this resolves to 'n' because of short-circuit evaluation...

Comment (ralf-meyer) [reply molSimplify/Scripts/inparse.py:486]: Had to run that snippet in order to figure out what it actually evaluates to. 😄

**Pull request merged**